.github/PULL_REQUEST_TEMPLATE/pr_docs.md

```
@@ -27,6 +33,34 @@ labels: ["status:needs-review", "lang:md"]
 
 - Sources/references: <links>
 
+## Changelog
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)